tool/convert_book.dart

```diff
@@ -22,7 +22,7 @@ convertFile(String fileName) {
   print(fileName);
   var file = new File(fileName);
   var contents = file.readAsStringSync();
-  var title = new RegExp(r'<title>(.*)</title>').firstMatch(contents)[1];
+  var title = new RegExp(r'<title>(.*)</title>').firstMatch(contents)?[1];
```

Comment:
It might make more sense to migrate this using a null check (`!`). It matches the previous behavior and avoids needing to make `title` nullable.

Reply:
This script is very old, and I don't find any references to it. Delete it?

Reply:
Sounds good to me, deleted!